apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -53,6 +53,9 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
   const { rescheduleUid } = router.query;
 
   const [brand, setBrand] = useState("#292929");
+  const [isSlotReservationPending, setIsSlotReservationPending] = useState(false);
+
+  const didSlotReservationFail = reserveSlotMutation.isError;
```

Comment:
Hey @roae , Thanks for the input and yes I tried that initially but it didn't work as I explained in the description of the PR at the top:
"I used a local state instead of the state from reservationSlotMutation because there is still some delay between the mutation completes and the links takes the user to the next page and during that delay, they can still select a different time slot by mistake."